# @author Maik Basso <maik@maikbasso.com.br>

Header header

int32 numberOfMessagesSent
int32 numberOfMessagesReceived
int32 numberOfTasksReceived
int32 numberOfAssignTasksForOtherWorkers
int32 numberOfCompletedTasks
